specs/lightclient/beacon-chain.md

```diff
@@ -81,6 +82,7 @@ The reward fractions add up to 7/8, leaving the remaining 1/8 for proposer rewar
 | - | - |
 | `PARTICIPATION_FLAGS_LENGTH` | `8` |
 | `G2_POINT_AT_INFINITY` | `BLSSignature(b'\xc0' + b'\x00' * 95)` |
+| `GWEI_PER_ETH` | `10**9` |
```

Comment:
((porting previous comments to here))

Maybe use `ether` instead of `eth` in the denomination context?

- `GWEI_PER_ETH` -> `GWEI_PER_ETHER`
- `get_base_reward_per_eth` -> `get_base_reward_per_ether`